diff -p -up evolution-sharp-0.21.1/configure.in.fixmajor evolution-sharp-0.21.1/configure.in
--- evolution-sharp-0.21.1/configure.in.fixmajor	2010-08-09 09:34:45.000000000 +0200
+++ evolution-sharp-0.21.1/configure.in	2010-08-09 09:38:12.000000000 +0200
@@ -33,11 +33,11 @@ elif test "x$edsver" = "x1.9" -o "x$edsv
 	EDATACAL_SOVERSION=6
 elif test "x$edsver" = "x2.24"; then
 	EDS_VERSION=-1.2
-	EDS_SOVERSION=11
-	EBOOK_SOVERSION=9
-	ECAL_SOVERSION=7
+	EDS_SOVERSION=14
+	EBOOK_SOVERSION=10
+	ECAL_SOVERSION=8
 	EBACKEND_SOVERSION=0
-	EDATACAL_SOVERSION=6
+	EDATACAL_SOVERSION=10
 elif test "x$edsver" = "xunsupported"; then
 	AC_MSG_ERROR([
 
